apps/site/assets/static/images/icon-change-direction.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
```

Comment:
Can we add a title to this? We did that for all of our other SVGs for accessibility (lol as you might remember)